debian/patches/revert-551f560d-cloud-config-after-snap-seeding.patch

```diff
@@ -88,14 +88,17 @@ Last-Update: 2024-02-14
      result = get_mount_info(mount_point, get_mnt_opts=True)
 --- a/systemd/cloud-config.service.tmpl
 +++ b/systemd/cloud-config.service.tmpl
-@@ -3,6 +3,7 @@
+@@ -2,8 +2,8 @@
+ [Unit]
  # https://cloudinit.readthedocs.io/en/latest/explanation/boot.html
  Description=Cloud-init: Config Stage
- After=network-online.target cloud-config.target
-+After=snapd.seeded.service
- Before=systemd-user-sessions.service
- Wants=network-online.target cloud-config.target
+-After=network-online.target cloud-config.target
+-Wants=network-online.target cloud-config.target
++After=network-online.target cloud-config.target snapd.seeded.service
++Wants=network-online.target cloud-config.target snapd.seeded.service
```

Comment:
Adding `snapd.seeded.service` here has no effect, right? It doesn't hurt anything, but I also don't see why it should be added.

Reply:
Agreed on both counts. I'll back that out on Monday having just missed this comment.

Reply:
Fixed. focal/jammy/mantic. We don't want that Wants= in there.